app/src/main/java/id/rizmaulana/covid19/data/source/pref/AppPrefSource.kt

```diff
@@ -15,7 +15,7 @@ class AppPrefSource {
 
     fun setOverview(covidOverview: CovidOverview) = Hawk.put(CacheKey.OVERVIEW, covidOverview)
 
-    fun getDaily(): List<CovidDaily>? = Hawk.get(CacheKey.DAYS, null)
+    fun getDaily(): List<CovidDaily>? = Hawk.get(CacheKey.DAYS, emptyList())
```

Comment:
We can just remove nullable data type here 
```suggestion
    fun getDaily(): List<CovidDaily> = Hawk.get(CacheKey.DAYS, emptyList())
```